backend/src/bundles/wallets/wallets.controller.ts

```diff
@@ -126,9 +127,20 @@ class WalletController extends Controller {
             throw new Error(WalletValidationMessage.TOKEN_REQUIRE);
         }
         const userId = getUserIdFromToken(token);
+        const wallets = await this.walletService.findAllWallets(userId);
+
+        const response: unknown[] = [];
+        for (const item of wallets.items) {
```

Comment:
i think we can fix this antipattern case with creating helper

Reply:
another branch https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/issues/415